### DISCUSSION TITLE
Question about performance

### DISCUSSION THREAD

**Discussion by SirCrocker** [thecoder-001/MineColab#89] [Q&A]
Hi, I found this project today and just wanted to ask how well does the server perform? 

Best Regards,
Agustin

**Comment by N-aksif-N** [comment]
Well, Its best to use with 5+ players. But with 10+ or 20+ players, you need to use plugins to reduce lag and improve performance. 

 If you don't might, try this: https://github.com/N-aksif-N/MineColab_Improved
It's a better improvement for minecolab
